AnkiDroid/src/main/java/com/ichi2/anki/model/DeckMetaDataModel.kt

```
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2022 Prateek Singh <prateeksingh3212@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.ichi2.anki.model
+
+import com.ichi2.utils.JSONException
+import com.ichi2.utils.JSONObject
+import org.intellij.lang.annotations.Language
+import timber.log.Timber
+
+data class DeckMetaDataModel(
+    val did: Long,
+    val deckName: String,
+    val new: Int,
+    val lrn: Int,
+    val rev: Int,
+    val eta: Int
+) {
+
+    @Language("JSON")
+    private val emptyData = """
+        {
+           "did":-1,
+           "deckName":"Ankidroid_Empty",
+           "new":0,
+           "lrn":0,
+           "rev":0,
+           "eta":0
+        }
+    """.trimIndent()
+
+    fun toJSON(): String {
+        val jsonObject = JSONObject()
+        try {
+            jsonObject.put("did", did)
+            jsonObject.put("deckName", deckName)
+            jsonObject.put("new", new)
+            jsonObject.put("lrn", lrn)
+            jsonObject.put("rev", rev)
+            jsonObject.put("eta", eta)
+        } catch (e: JSONException) {
```

This shouldn't be possible, let the exception be raised if this is the case.

---

Done 👍🏽